js/src/annotations/osd-svg-overlay.js

```diff
@@ -386,6 +131,265 @@
     listenForActions: function() {
       var _this = this;
 
+      this._thisResize = function(){
+        _this.resize();
+      };
+      this._thisRotate = function(event) {
+        _this.rotate(event);
+        _this.resize();
+      };
+      this.viewer.addHandler('animation', this._thisResize);
+      this.viewer.addHandler('open', this._thisResize);
+      this.viewer.addHandler('animation-finish', this._thisResize);
+      this.viewer.addHandler('update-viewport', this._thisResize);
+      this.viewer.addHandler('resize',this._thisResize);
+      this.viewer.addHandler('rotate', this._thisRotate);
+      this.viewer.addHandler('constrain',this._thisResize);
+
+      this._thisDestroy = function(){
+        _this.destroy();
+      };
+
+      this.viewer.addHandler('close',this._thisDestroy);
+
+      this.eventsSubscriptions.push(this.eventEmitter.subscribe('DESTROY_EVENTS.'+this.windowId, function(event) {
```

Comment:
I don't understand.  Since it's part of the `eventsSubscriptions` array, it automatically gets unsubscribed on the next initialization (as part of `destroy), and then gets re-subscribed